### ISSUE TITLE
Failed image url

### ISSUE THREAD

**Issue by jasonlaw** [ch-muhammad-adil/flutter-circular_profile_avatar#1]
Hi @ch-muhammad-adil ,

What would happen if the image url is not valid? Will there be a default empty/invalid image display? If not, do you think it is worth to put on it?

Thanks in advance.

**Comment by ch-muhammad-adil** [comment]
I will see all available options, and fix this, thank you for creating issue.

**Comment by ch-muhammad-adil** [comment]
Fixed in version 0.1.9 
It will work  #if cacheImage is true. 

```
   CircularProfileAvatar(    
               'https/avatars0.github', // invalid URL, 
                errorWidget: (context, url, error) => Container(
                      child: Icon(Icons.error),
                    ),
                placeHolder: (context, url) => Container(
                      width: 50,
                      height: 50,
                      child: CircularProgressIndicator(),
                    ),
                cacheImage: true,
            .........
   )
```



in case of empty URL please try user's initials as well
